intelmq/bots/collectors/rsync/collector_rsync.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import os
+from intelmq.lib.bot import CollectorBot
+
+
+class RsyncCollectorBot(CollectorBot):
+    def process(self):
+        self.logger.info("Updating file {}.".format(self.parameters.file))
+        os.system("rsync {}/{} {}".format(self.parameters.rsync_path, self.parameters.file, os.path.dirname(__file__)))
```

Comment:
You are copying the file to the directory where the collector file resides. That may work for development environments, but `/usr/lib/` is not writeable for the intelmq user. Use the var/lib/bots path for state data of bots. I pushed a commit with which you can use `intelmq.VAR_STATE_PATH` to access this directory: 5611e059a4d5373dc06078844dfdfde7e7a73068

Reply:
Should i store data for rsync-collector directly in var/lib/bots or should i store data in var/lib/bots/rsync_collector?  I tried create directory var/lib/bots/rsync_collector in init method of rsync-collector, but tests failed becouse of permission.